pulp_file/tests/functional/api/test_crud_content_unit.py

```diff
@@ -180,6 +181,14 @@ def test_cannot_create_repo_version_with_two_relative_paths_the_same(
         monitor_task(response.task)
 
 
+@pytest.mark.parallel
+def test_bad_inputs_to_modify_endpoint(file_repo, file_repo_api_client, file_random_content_unit):
+    body = {"add_content_units": file_random_content_unit}
```

Comment:
This is testing with a dict instead of a list of strings, right?
How about 
```suggestion
    body = {"add_content_units": {}}
```
Maybe also test:
- `[{}]`
- `["/content/"]`
- `[f"{api_base_path}/contents/file/file/abcd"]`

Reply:
Great suggestion, thanks!